.github/workflows/create-release.yml

```diff
@@ -16,7 +16,7 @@ on:
         description: 'Draft'
         required: false
         default: 'false'
-        
```

Comment:
This file has only whitespace changes. I'm fine to revert it to simplify this PR.